internal/authn/oidc/oidc.go

```diff
@@ -83,26 +83,29 @@ func (oidc *RemoteOidcAuthenticator) Authenticate(requestContext context.Context
 	if !token.Valid {
 		return nil, errInvalidToken
 	}
-
 	claims, ok := token.Claims.(jwt.MapClaims)
 	if !ok {
 		return nil, errInvalidClaims
 	}
 
-	validIssuers := []string{
-		oidc.MainIssuer,
+	if err := jwt.NewValidator(jwt.WithIssuedAt()).Validate(claims); err != nil {
+		return nil, errInvalidToken
 	}
+
+	validIssuers := []string{oidc.MainIssuer}
```

Comment:
nitpick: this line can be combined with the next line.

Reply:
changed to `validIssuers := append(oidc.IssuerAliases, oidc.MainIssuer)`

is it ok?